examples/cloud-router-2-oracle-connection/versions.tf

```diff
@@ -5,5 +5,9 @@ terraform {
       source  = "equinix/equinix"
       version = ">= 1.20.0"
```

Comment:
You need to change the version for all examples to use the version that includes your WaitForDeprovision changes. The versions before that won't work for these examples.